tests/test_client.py

```diff
@@ -149,41 +156,41 @@ def test_proxy_httpsselect_bothenv_http(monkeypatch):
     assert client.transport._pool.proxy.scheme == "http"
 
 
-def test_simple_transport():
+def test_simple_transport(sentry_init):
     events = []
```

Comment:
You do not need to pass in a transport if you're using sentry_init, just use the `capture_events` fixture instead which does the same thing.

Reply:
I did that at first but then decided that wasn't in the spirit of the original test and the name `test_simple_transport`.

Reply:
ah well you're right